static/app/views/starfish/utils/useEventsStatsQuery.tsx

```diff
@@ -39,12 +39,14 @@ export function useEventsStatsQuery({
     }),
     options: {
       enabled,
+      retry: false,
```

Comment:
> retry: boolean | number | (failureCount: number, error: TError) => boolean
If false, failed queries will not retry by default.
If true, failed queries will retry infinitely.
If set to a number, e.g. 3, failed queries will retry until the failed query count meets that number.

setting retry to false because it retries a couple times on timeout and that almost never succeeds and it's just a lot of loading until then which is super annoying.